common/djangoapps/enrollment/data.py

```diff
@@ -293,3 +296,23 @@ def get_course_enrollment_info(course_id, include_expired=False):
         raise CourseNotFoundError(msg)
     else:
         return CourseSerializer(course, include_expired=include_expired).data
+
+def get_roster(course_id):
+    roster = []
+
+    course_key = CourseKey.from_string(course_id)
+    qset = CourseEnrollment.objects.filter(
```

Comment:
I know existing code writes `qset` everywhere, but I prefer we be more explicit here and call this "queryset". But even then, that the underlying data structure is a queryset is probably irrelevant to most developers; they only care that it's an iterable collection of elements.

We should just call this enrollments, which lends itself to the later iteration like:
```
roster = [
    enrollment
    for enrollment in enrollments
]
```